<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sucesso</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="tela_sucesso">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-5</y>
      <width>410</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="sucessso_titulo">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>400</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(31, 106, 165);
background-color: rgb(43, 43, 43);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;SAIR DO WINDOWS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sucesso_sub">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(36, 147, 58);
font-weight: bold;
font-size: 30px</string>
     </property>
     <property name="text">
      <string>SUCESSO!</string>
     </property>
    </widget>
    <widget class="QLabel" name="verificado">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(36, 147, 58);
font-weight: bold;
font-size: 30px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/icon/Documents/Programacao/MeusProjetos/logoff/version.2.2/icons8-verificado-96.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="informacao_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 25px;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seu pc será desligado em:&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="check">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>400</y>
       <width>196</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font-size: 15px</string>
     </property>
     <property name="text">
      <string>Bloquear o usuário ao sair</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_finalizar">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>450</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(31, 106, 164);
color: rgb(255, 255, 255);
font-weight: bold;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>Até mais!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelar">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>450</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);
font-weight: bold;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLabel" name="informacao_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 25px;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Até mais!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="informacao_h">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 25px;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;00 : 00 : 00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/icons.qrc"/>
 </resources>
 <connections/>
</ui>
